packages/backend/src/api/controllers/TransactionSubmitController.ts

```diff
@@ -70,7 +76,7 @@ export class TransactionSubmitController {
       data: {
         type: 'Withdraw',
         starkKey: data.starkKey,
-        assetType: data.assetType,
+        assetType: AssetHash(data.assetType),
```

Comment:
The `assetType` name is really confusing. At least for me. If it's always an AssetHash we should probably change I to that

Reply:
I agree, but that's also how Starkware names them. It's AssetType for ERC and ETH, and AssetId for Perpetual and for Spot NFT, but also there types are different (id or hash). I agree we should unify it more on our side, but at the explorer-ethereum touchpoint we will need to use Starkware names from solidity.